### ISSUE TITLE
Button fails WCAG 2.1 AAA contrast ratio

### ISSUE THREAD

**Issue by WilliamKelley** [prenda-school/design-system#220]
All variants of `Button` fail to meet the WCAG 2.1 AAA contrast ratio for normal text. See the screenshot at bottom.

Of course one option is changing every variant to meet AAA guidelines. However, another option is providing a "high contrast" version of each variant. A user needing AAA contrast would be able to "turn on" a high contrast "mode" for the website (design system) and all components would change accordingly. The standard for reference: https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

I think this issue shows the need for broader consideration of how Spark Web components use text. While the font attributes are taken from Typography presets, most font _color_ is not. White (#FFFFFF) is commonly used but that color isn't even sanctioned for fonts by the Typography standard. This should trigger a major revision of the Typography standard to expand and advise all use-cases and refactor of web components to only implement advised use-cases. Or where need-be, ensure that alternatives meet standard criteria.

![image](https://user-images.githubusercontent.com/25781782/130998009-c06b3d10-cbf5-429e-a698-be26d1d159a5.png)

**Comment by alecdanz** [comment]
Can be closed #450
